server/index.js

```diff
@@ -37,8 +69,41 @@ app.prepare().then(() => {
     });
   });
 
+  server.delete('/_api/content/remove-media', (req, res) => {
+    return content.removeMedia(req.query.name, result => {
+      res.send(result);
+    });
+  });
+
+  server.post('/_api/content/save-media', (req, res) => {
+    const promise = new Promise((resolve, reject) => {
+      upload.array('file', 12)(req, res, err => {
```

Comment:
what is `12` exactly? can you define a constant for that?

Reply:
Max files count